pkg/errors/errors.go

```diff
@@ -106,3 +107,20 @@ func IsLaunchTemplateNotFound(err error) bool {
 	}
 	return false
 }
+
+func IsRefreshCredentials(err error) bool {
+	if err == nil {
+		return false
+	}
+	if strings.Contains(err.Error(), "failed to refresh cached credentials") {
```

Comment:
What's the code on this? Generally I'm not sure what the guarantees are on the formatting of this messaging that I worry about relying on it.